pkg/k8s/apis/cilium.io/v2alpha1/bgpp_types.go

```diff
@@ -9,6 +9,19 @@ import (
 	slimv1 "github.com/cilium/cilium/pkg/k8s/slim/k8s/apis/meta/v1"
 )
 
+const (
+	// DefaultBGPConnectRetryTimeSeconds defines the default initial value for the BGP ConnectRetryTimer (RFC 4271, Section 8).
+	DefaultBGPConnectRetryTimeSeconds = 120
+	// DefaultBGPHoldTimeSeconds defines the default initial value for the BGP HoldTimer (RFC 4271, Section 4.2).
+	DefaultBGPHoldTimeSeconds = 90
+	// DefaultBGPKeepAliveTimeSeconds defines the default initial value for the BGP KeepaliveTimer (RFC 4271, Section 8).
+	DefaultBGPKeepAliveTimeSeconds = 30
+	// DefaultBGPGRRestartTimeSeconds defines default Restart Time for graceful restart (RFC 4724, section 4.2)
+	DefaultBGPGRRestartTimeSeconds = 120
+	// DefaultBGPPeerPort defines the TCP port number of a CiliumBGPNeighbor when PeerPort is unspecified.
+	DefaultBGPPeerPort = 179
+)
+
```

Comment:
What is still missing is the type change of: 
```go
LocalASN int `json:"localASN"

PeerASN int `json:"peerASN"`
```

I think these should become `int64` to allow for maximum value `4294967295`.
(`uint32` would be best, but it seems unsigned integers are not really used in k8s API)

Reply:
But note that that will be a bigger change, as ASNs are used in various data structures in the backend, where they are maniupulated as `int` (which actually could be an issue on 32bit systems, if Cilium would run on those)

Reply:
Done in https://github.com/cilium/cilium/pull/26040/commits/a811886ed98a9f856105b0fa35c267f4d12be357